Makefile

```diff
@@ -36,7 +36,7 @@ remove-install-stamp:
 update: remove-install-stamp install ## Update the dependencies
 
 .PHONY: serve
-serve: install ## Run the ihatemoney server
+serve: install build-translations ## Run the ihatemoney server
```

Comment:
I'm wondering: does this mean it will compile the translations everytime we serve? Do we have a way to cache this maybe?

Reply:
It can probably detect wether `*.mo` files exists before compiling catalogs. But catalog compilation is fast enough and can be silented with `--quiet`. What do you think?

Reply:
I'll trust you here : If you think the compilation is fast enough, then I could live with it :-)